Applications/Spire/Source/TimeAndSalesUiTester/main.cpp

```diff
@@ -1,18 +1,197 @@
 #include <QApplication>
+#include "Spire/Spire/ArrayListModel.hpp"
 #include "Spire/Spire/Dimensions.hpp"
 #include "Spire/Spire/Resources.hpp"
-#include "Spire/TimeAndSales/TimeAndSalesPropertiesWindowFactory.hpp"
+#include "Spire/TimeAndSales/TimeAndSalesTableView.hpp"
+#include "Spire/TimeAndSalesUiTester/DemoTimeAndSalesModel.hpp"
+#include "Spire/Ui/CustomQtVariants.hpp"
+#include "Spire/Ui/DropDownBox.hpp"
+#include "Spire/Ui/IntegerBox.hpp"
+#include "Spire/Ui/MoneyBox.hpp"
+#include "Spire/Ui/TextBox.hpp"
 #include "Version.hpp"
 
+using namespace boost::posix_time;
+using namespace Nexus;
 using namespace Spire;
 
+auto to_text(BboIndicator indicator) {
```

Comment:
This function is useful, make it part of the ```BboIndicator```'s API (not a method, just a free function declared in the same header file as ```BboIndicator```.